toolchain/toolchain/python/innercoremodtoolchain/utils.py

```diff
@@ -162,6 +163,6 @@ def _extract_member(self, member, targetpath, pwd):
 			targetpath = super()._extract_member(member, targetpath, pwd)
 
 			attr = member.external_attr >> 16
-			if attr != 0:
+			if attr != 0 and not isdir(targetpath):
 				os.chmod(targetpath, attr)
```

Comment:
Mostly likely reason is attempting to change flags on parent directory when extracting it, which also denies to working with anything inside here.